include/hipSYCL/sycl/handler.hpp

```diff
@@ -437,7 +437,7 @@ void set_args(Ts &&... args);
                                         data_src};
     // Assume dest element size and allocation shape is the same as src
```

Comment:
I think that this comment is now not correct anymore. Can you adjust or remove it?